g2p_registry_rest_api_extension_demo/models/group.py

```diff
@@ -0,0 +1,5 @@
+from odoo.addons.g2p_registry_rest_api.models.group import GroupInfoOut
+
+
+class GroupInfoOutExtended(GroupInfoOut, extends=GroupInfoOut):
+    z_ind_grp_num_children: int
```

Comment:
Replace by another field so we do not have to require the installation of the demo to run this demo.